content/docs/start/data-and-model-access.md

```diff
@@ -111,8 +111,8 @@ directly from within an application at runtime. For example:
 import dvc.api
 
 with dvc.api.open(
-        'get-started/data.xml',
-        repo='https://github.com/iterative/dataset-registry'
-        ) as fd:
-    # ... fd is a file descriptor that can be processed normally.
+    'get-started/data.xml',
+    repo='https://github.com/iterative/dataset-registry'
+) as fd:
+    # fd is a file descriptor which can be used here
```

Comment:
Why remove the "processed normally" part? And period
```suggestion
    # fd is a file descriptor, which can be processed normally.
```

Reply:
honestly this just need to say `fd is a file descriptor`. A code comment isn't meant to be prosaic.

Reply:
It's not a regular code commit you would see in the actual code. The actual code would just use `fd`. This is meant to replace multiple lines of code that are irrelevant for explaining the DVC API. The point is now you have a file descriptor that you can use normally, as you would use any file descriptor.

That said I'd be OK with just removing the 2nd half since it's kind of obvious for anyone who knows what a FD is what you'd do with it. But I fear again we're not focusing on impactful things.

Reply:
changed